src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationType.java

```diff
@@ -22,7 +22,8 @@ public enum AcEquationType implements Quantity {
     BRANCH_TARGET_Q("branch_target_q", ElementType.BRANCH), // generator reactive power control
     BRANCH_TARGET_ALPHA1("branch_target_\u03B1", ElementType.BRANCH), // phase shifter constant shift
     BRANCH_TARGET_RHO1("branch_target_\u03C1", ElementType.BRANCH), // transformer constant voltage control
-    DISTR_Q("distr_q", ElementType.BUS), // remote voltage control reactive power distribution
+    DISTR_Q_VOLTAGE_CONTROL("distr_q_voltage_control", ElementType.BUS), // remote voltage control reactive power distribution
```

Comment:
You have doubled all the code but the logic is very similar between distribution equations for voltage control and for reactive power remote control. I am quite convinced that everything can be mutualized with a deep understanding of both feature.